### PR TITLE
chore: remove global loading spinner

### PR DESCRIPTION
**Pull request by alaister**
## What is the current behaviour?

A big ol' spinner before the app is visible makes it impossible to implement skeleton UI.

## What is the new behaviour?

Removes the global loading spinner when the user first visits the dashboard.

## Additional context

This is using the prerelease version of `@supabase/supabase-js`, which I feel okay about dogfooding on the dashboard, but if you have a different opinion, please share!

Due to it using the prerelease version, it should also prevent users from getting logged out so much.

## Testing

I think I ran into some states where the loading spinners got stuck, but I couldn't easily reproduce. This needs to be thoroughly tested before getting merged.

### CODE REVIEW

**Comment by saltcod** [comment]
> I think I ran into some states where the loading spinners got stuck

@alaister do you remember where? I just tested a bunch and didn't run into anything.

**Comment by alaister** [comment]
@saltcod It seems to be _very_ intermittent, which is annoying.
Let's hold off on merging this until `gotrue-js` v2 is stable. I still need to do some more testing in gotrue land to make sure the issue isn't there!

File: studio/components/layouts/AccountLayout/WithSidebar.tsx [hunk 1]
```diff
@@ -86,8 +86,8 @@ const WithSidebar: FC<Props> = ({
 export default WithSidebar
 
 const LinksWithHeaders: FC<any> = ({ links, subitems, subitemsParentKey }) => {
-  return links.map((x: any) => (
-    <div key={x.heading} className="dark:border-dark border-b py-5 px-6">
+  return links.map((x: any, i: number) => (
+    <div key={x.heading || `${i}`} className="dark:border-dark border-b py-5 px-6">
```
Comment (joshenlim) [inline studio/components/layouts/AccountLayout/WithSidebar.tsx:90]: any reason why we need to do this? is there a link that has no heading?

Comment (alaister) [reply studio/components/layouts/AccountLayout/WithSidebar.tsx:90]: This was just to remove an annoying error in the console. I think there must have been a link with no heading causing duplicate key errors. This was just a quick fix.
Happy to investigate further if you think there's a benefit/cleaner code?

Comment (joshenlim) [reply studio/components/layouts/AccountLayout/WithSidebar.tsx:90]: ahh okay no worries i can get into this while i test

**Comment by joshenlim** [comment]
All looks good - will merge it on Monday

**Review by joshenlim — APPROVED** [review]

**Pull request merged**